examples/version-migration/separate-container/tests/separateContainer.test.ts

```diff
@@ -121,27 +122,28 @@ describe("separate-container migration", () => {
 				const migrationStatusElements = document.querySelectorAll(".migration-status");
 				return migrationStatusElements[1]?.textContent?.includes("one") === true;
 			});
-			await expect(leftContainsOne).toEqual(true);
-			await expect(rightContainsOne).toEqual(true);
+			expect(leftContainsOne).toEqual(true);
+			expect(rightContainsOne).toEqual(true);
 
 			// Wait until we see the summarizer join
-			await page.evaluate(() => {
+			await page.evaluate(async () => {
 				// This is reaching a bit, but we just need to watch it for test purposes.
 				const leftQuorum = (
-					window["migrators"][0]._currentMigratable.model.container as IContainer
-				).getQuorum();
-				const alreadyHasSummarizer =
-					[...leftQuorum.getMembers().values()].find(
-						(client) => client.client.details.type === "summarizer",
-					) !== undefined;
+					(window["migrators"] as IMigrator[])[0].currentModel as unknown as {
+						container: IContainer;
+					}
+				).container.getQuorum();
+				const alreadyHasSummarizer = [...leftQuorum.getMembers().values()].some(
+					(sequencedClient) => sequencedClient.client.details.type === "summarizer",
+				);
 				if (alreadyHasSummarizer) {
 					// This should be the path taken since demo mode should spawn the summarizer instantly.
 					return;
 				}
 				// In case the summarizer isn't quite connected yet, return a Promise so we can await for it to join.
 				return new Promise<void>((resolve) => {
-					const watchForSummarizer = (clientId, details) => {
-						if (details.type === "summarizer") {
+					const watchForSummarizer = (clientId, sequencedClient: ISequencedClient): void => {
+						if (sequencedClient.client.details.type === "summarizer") {
```

Comment:
This was an actual bug, but unlikely to be the source of flakiness (otherwise we would expect timeout errors).